### DISCUSSION TITLE
notes/MathematicsAndPhysics/LinearAlgebra/

### DISCUSSION THREAD

**Discussion by giscus** [shrike-505/Nest#11] [Announcements]
# notes/MathematicsAndPhysics/LinearAlgebra/

https://shrike505.cc/notes/MathematicsAndPhysics/LinearAlgebra/

**Comment by Shuixin-BMateGamer** [comment]
同志, SVD
$$M=Q\Sigma P^*$$

**Reply by shrike-505** [reply]
你还是出去站一会吧
